### PR TITLE
Add Sift's user-case

### PR DESCRIPTION
**Pull request by aijamalnk**

### CODE REVIEW

**Comment by aijamalnk** [comment]
@mik-laj  can you please take a look at this?

**Comment by mik-laj** [comment]
@mschickensoup  Do you have time to look at it?

**Comment by mschickensoup** [comment]
I should be available to take a look at it around Friday, would it work? @mik-laj

File: landing-pages/site/content/en/use-cases/sift.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+##### How did Apache Airflow help to solve this problem?
+
+Airflow makes it easy to clearly define the interactions between various jobs, expanding the scope of what we can do in our model training pipelines. We now have the ability to schedule and coordinate all jobs while managing the dependencies between them using DAGs. Each of our main workflows, including our model training pipeline and our ETL pipelines, has its own DAG code that manages its tasks’ dependencies and the execution schedule for the pipeline. We even define dependencies between separate DAGs by using Airflow’s ExternalTaskSensor. This allows our DAGs to actually depend on each other and allows us to keep each one focused and compact in its scope.
```
Comment (mschickensoup) [inline landing-pages/site/content/en/use-cases/sift.md:20]:
```suggestion
Airflow makes it easy to clearly define the interactions between various jobs, expanding the scope of what we can do in our model training pipelines. We now have the ability to schedule and coordinate all jobs while managing the dependencies between them using DAGs. Each of our main workflows, including our model training pipeline and  ETL pipelines, has its own DAG code that manages its tasks’ dependencies and the execution schedule for the pipeline. We even define dependencies between separate DAGs by using Airflow’s ExternalTaskSensor. This allows our DAGs to actually depend on each other and keep each one of them focused and compact in its scope. 
```

**Comment by aijamalnk** [comment]
Sorry i missed this! @mik-laj @mschickensoup is there anything else i need to do? Thank you both!

**Comment by mik-laj** [comment]
CI was sad, but I've already fixed it.

**Pull request merged**